config/server-vars.yml

```diff
@@ -100,6 +100,7 @@ EDXAPP_FEATURES:
 EDXAPP_COMPREHENSIVE_THEME_DIRS: %%EDXAPP_COMPREHENSIVE_THEME_DIRS%%
 EDXAPP_ENABLE_COMPREHENSIVE_THEMING: %%EDXAPP_ENABLE_COMPREHENSIVE_THEMING%%
 EDXAPP_DEFAULT_SITE_THEME: "%%EDXAPP_DEFAULT_SITE_THEME%%"
+COMPREHENSIVE_THEME_DIRS: "{{ EDXAPP_COMPREHENSIVE_THEME_DIRS }}"
```

Comment:
Removing that line breaks the deployment. At least that's what I saw. Did you see otherwise?